tests/phpunit/integration/tests/REST_API/Hotlinking_Controller.php

```diff
@@ -119,6 +120,16 @@ public function mock_http_request( $preempt, $r, $url ) {
 			];
 		}
 
+		if ( self::URL_MALFORMED_MIME === $url ) {
```

Comment:
Malformed is the wrong word.

`image/png; charset=utf-8` is a totally legitimate `Content-Type` header. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type.

I'd rename it to something like "content-type with mime type" or so